gateleen-routing/src/main/resources/gateleen_routing_schema_routing_rules

```diff
@@ -132,6 +136,32 @@
 			],
 			"additionalProperties": false
 		},
+		"ProxyOptions": {
+			"properties": {
+				"type": {
+                    "type": "string",
+                    "enum": [
+                        "HTTP",
+                        "SOCKS4",
+                        "SOCKS5"
+                    ]
+				},
+				"host": {
+					"type": "string"
+				},
+				"port": {
+					"type": "integer",
+					"minimum": 0
+				},
+				"username": {
+					"type": "string"
+				},
+				"password": {
+					"type": "string"
+				}
+			},
+			"additionalProperties": false
+		},
```

Comment:
**Bad indentation:** Some of those lines are indent by _4 spaces_ where rest of file uses _1 tab_.